service/src/test/java/com/ge/predix/acs/policy/evaluation/cache/AbstractPolicyEvaluationCacheTest.java

```diff
@@ -441,24 +439,25 @@ public void testHaveConnectorIntervalsLapsed(final AttributeConnector resourceCo
         request.setAction(ACTION_GET);
         request.setSubjectIdentifier(AGENT_MULDER);
         request.setResourceIdentifier(XFILES_ID);
+        request.setPolicySetsEvaluationOrder(EVALUATION_ORDER_POLICYONE);
         PolicyEvaluationRequestCacheKey key = new PolicyEvaluationRequestCacheKey.Builder().zoneId(ZONE_NAME)
                 .request(request).build();
 
         PolicyEvaluationResult result = mockPermitResult();
         spiedCache.set(key, result);
         spiedCache.get(key);
-        Mockito.verify(spiedCache, Mockito.times(isResourceConnectorConfigured || isSubjectConnectorConfigured ? 0 : 1))
-                .havePrivilegeServiceAttributesChanged(Mockito.any(), Mockito.any());
+        Mockito.verify(spiedCache, Mockito.times(isResourceConnectorConfigured || isSubjectConnectorConfigured ? 1 : 2))
+                .haveEntitiesChanged(Mockito.any(), Mockito.any());
         Mockito.verify(spiedCache, Mockito.times(isResourceConnectorConfigured || isSubjectConnectorConfigured ? 1 : 0))
                 .haveConnectorCacheIntervalsLapsed(Mockito.any(), Mockito.any());
         Assert.assertEquals(this.cache.haveConnectorCacheIntervalsLapsed(connectorService, currentTime),
                 haveConnectorCacheIntervalsLapsed);
 
     }
 
-    public static PolicyEvaluationResult mockPermitResult() {
+    private static PolicyEvaluationResult mockPermitResult() {
         PolicyEvaluationResult result = new PolicyEvaluationResult(Effect.PERMIT);
-        result.setResolvedResourceUris(new HashSet<>(Arrays.asList(new String[] { XFILES_ID })));
+        result.setResolvedResourceUris(new HashSet<>(Collections.singletonList(XFILES_ID)));
```

Comment:
Use `Collections.singleton(..)` instead of `new HashSet<>(Collections.singletonList(..))`